src/e2e/app/repo-tests/src/test_flight_sql.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use indoc::indoc;
+use kamu_cli_e2e_common::KamuApiServerClientExt;
+use kamu_node_e2e_common::KamuFlightSQLClient;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub async fn test_flight_sql_self_test(mut kamu_flight_sql_client: KamuFlightSQLClient) {
+    kamu_flight_sql_client.set_anonymous().await;
+
+    kamu_flight_sql_client
+        .flight_sql_assert_call(
```

Comment:
I think it would be a good idea to check the result of the execution as well

Reply:
But in the `test_flight_sql()` &amp; `test_flight_sql_anonymous()` tests, you do check the result from the server, right?

And here you pass `None' as the third argument, why?